commands/VariableCommand.js

```diff
@@ -272,6 +278,11 @@ VariableCommand.prototype = extend(BaseCommand.prototype, {
 		}
 		this.checkArguments(arguments);
 
+		if(variableName.length > this.maxNameLength) {
```

Comment:
`deviceId` can also be a variable name at this point, so you should add another check inside of `disambiguateGetValue`.